languages.toml

```
@@ -880,6 +880,28 @@ indent = { tab-width = 4, unit = "    " }
 name = "php"
 source = { git = "https://github.com/tree-sitter/tree-sitter-php", rev = "f860e598194f4a71747f91789bf536b393ad4a56" }
 
+[[language]]
+name = "php_only"
```

```suggestion
name = "php-only"
```

Also the `runtime/queries/php_only` dir should be renamed as `runtime/queries/php-only`. That should fix the injected php content

---

Yup, I was grepping left and right, thanks! :smile: